app/js/overview.js

```diff
@@ -0,0 +1,33 @@
+document.querySelector(".overview").addEventListener("click", function() {
+    const tableBody = document.querySelector('.overview_table_body')
+    // Sets the table body content as empty in case any table body has been created before
+    tableBody.innerHTML = ''
+    let overviewData = []
+    let numOfQuestions = document.querySelectorAll('#questions .question').length
+    for (let i = 1; i <= numOfQuestions; i++) {
```

Comment:
Using a for loop in here because im not looping over an array, i just use array.length as my total iteration count, plus if i use a forEach i need to change all of the `i`s to `i + 1` to give the same effect i needed, it's less readable that way.